falco/README.md

```diff
@@ -139,6 +139,19 @@ The following table lists the configurable parameters of the Falco chart and the
 | `affinity`                                      | The affinity constraint                                                                                            | `{}`                                                                                                                                      |
 | `tolerations`                                   | The tolerations for scheduling                                                                                     | `node-role.kubernetes.io/master:NoSchedule`                                                                                               |
 | `scc.create`                                    | Create OpenShift's Security Context Constraint                                                                     | `true` 
+`                                                                                                                                       |
+| `eksCloudwatch.enabled`                           | Deploy the forwarding agent for cloudwatch auditing logs, falco.webserver and auditlog must be enabled!                                                                 | `false `                                                                                                                                      |
+| `eksCloudwatch.image.registry`                                | The image registry to pull from                                                                                    | `docker.io`                                                                                                                               |
+| `eksCloudwatch.image.repository`                              | The image repository to pull from                                                                                  | `falcosecurity/falco`                                                                                                                     |
+| `eksCloudwatch.image.tag`                                     | The image tag to pull                                                                                              | `0.21.0`                                                                                                                                  |
```

Comment:
It seems to me that default values are different:

```
image:
    registry: docker.io
    repository: sysdiglabs/k8sauditlogforwarder
    tag: ekscloudwatch-0.2
```

Could you fix that please?

Reply:
Done

Reply:
I still see:

```
| `eksCloudwatch.image.repository`                              | The image repository to pull from                                                                                  | `falcosecurity/falco`                                                                                                                     |
| `eksCloudwatch.image.tag`                                     | The image tag to pull                                                                                              | `0.2.0`
```